admin/server/reports.go

```diff
@@ -769,13 +792,25 @@ type reportYAML struct {
 }
 
 type reportAnnotations struct {
-	AdminOwnerUserID string `yaml:"admin_owner_user_id"`
-	AdminManaged     bool   `yaml:"admin_managed"`
-	AdminNonce       string `yaml:"admin_nonce"` // To ensure spec version gets updated on writes, to enable polling in TriggerReconcileAndAwaitReport
-	WebOpenPath      string `yaml:"web_open_path"`
-	WebOpenState     string `yaml:"web_open_state"`
+	AdminOwnerUserID string      `yaml:"admin_owner_user_id"`
+	AdminManaged     bool        `yaml:"admin_managed"`
+	AdminNonce       string      `yaml:"admin_nonce"` // To ensure spec version gets updated on writes, to enable polling in TriggerReconcileAndAwaitReport
+	WebOpenPath      string      `yaml:"web_open_path"`
+	WebOpenState     string      `yaml:"web_open_state"`
+	WebOpenMode      WebOpenMode `yaml:"web_open_mode,omitempty"`
+	Explore          string      `yaml:"explore,omitempty"`
+	Canvas           string      `yaml:"canvas,omitempty"`
 }
 
+type WebOpenMode string
+
+const (
+	WebOpenModeRecipient WebOpenMode = "recipient"
+	WebOpenModeCreator   WebOpenMode = "creator"
+	WebOpenModeNone      WebOpenMode = "none"
+	WebOpenModeFiltered  WebOpenMode = "filtered"
+)
```

Comment:
This enum is both defined here in Go and defined in the protos. Can we use just one or the other type? IMO using just the Go enum seems simpler to me since we anyway need to store/parse it as a string. You can add a util function here for validating it when the frontend passes it as input:
```go
func (m WebOpenMode) Valid() bool {
  switch m {
  case WebOpenModeRecipient, WebOpenModeCreator, WebOpenModeNone, WebOpenModeFiltered:
    return true
  }
  return false
}
```